src/components/MultiHopTrade/components/MultiHopTradeConfirm/components/FeeStep.tsx

```diff
@@ -32,7 +32,11 @@ export const FeeStep = ({ isLastStep }: FeeStepProps) => {
   const inputAmountUsd = useAppSelector(selectInputSellAmountUsd)
   // use the fee data from the actual quote in case it varies from the theoretical calculation
   const affiliateBps = useAppSelector(selectActiveQuoteAffiliateBps)
-  const amountAfterDiscountUsd = useAppSelector(selectQuoteFeeAmountUsd)
+
+  const feeModel = 'SWAPPER'
+  const { feeUsd: amountAfterDiscountUsd } = useAppSelector(state =>
+    selectCalculatedFees(state, { feeModel, inputAmountUsd }),
```

Comment:
this filter prop will need a usememo

Reply:
https://github.com/shapeshift/web/pull/6503/commits/41da1997b88e26106bd870ade3a8b3430f13e04e